pkg/monitoring/hyperconverged/rules/alerts/operator_alerts.go

```diff
@@ -79,5 +81,17 @@ func operatorAlerts() []promv1.Rule {
 				healthImpactAlertLabelKey: "critical",
 			},
 		},
+		{
+			Alert: unsupportedArchitecturesAlert,
+			Expr:  intstr.FromString("kubevirt_hco_dict_with_no_supported_architectures == 1"),
```

Comment:
@nunnatsa  I think we should update this metric to `kubevirt_hco_dict_supported_architectures` and have the expression as `count(kubevirt_hco_dict_with_no_supported_architectures) == 0`.

Or we might even want to report this per node.

Reply:
I think that since we have limited and low number of architectures I would go with `kubevirt_hco_dataimportcrontemplate_architectures`, that provide the list of architectures per DICT.

and alert when `sum by (template) ( kubevirt_hco_dataimportcrontemplate_architectures) == 0`
and add the template name as a label.
So you can tell the user which templates have the issue.

Reply:
> @nunnatsa I think we should update this metric to either:
> 
> * `kubevirt_hco_dataimportcrontemplate_supported_architectures_count` and have the expression as `kubevirt_hco_dataimportcrontemplate_supported_architectures_count == 0`.
> 
> or
> 
> * `kubevirt_hco_dataimportcrontemplate_supported_architectures` that reports also the `architecture` label.

Not sure I follow. As far as I see it, there is no counting here - it's a binary metric - either the DICT has supported architectures in the cluster or it has not. The number of supported architecture does not serve us, IMHO.